projects/plugins/jetpack/changelog/update-changelog-952

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: compat
+Comment: Changelog: this is the changelog. :)
```

Comment:
Heh. It does ignore CHANGELOG.md when deciding whether to require a change file, but not readme.txt.